11ty/definitions/digital-blackface.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Digital Blackface
+slug: digital-blackface
+defined: true
+Speech: noun
+---
+
+Digital Blackface is a form of appropriation where white and non-black individuals use memes featuring Black people. This is a 21st century version of Black minstrelsy which was prevalent in the 19th and 20th centuries.
+
+
+## Issues
+
+ This type of action is another in a long history of dehumanizing black individuals into exaggerated versions of themselves, especially black women. Rarely are Black characters given subtle feelings, emotion or actions.
+
+
+## Examples
+It can go further than appropration where entire online personas are created. These are easy to spot as they almost always use terms like "as a black woman..." and resort to incorrect and illegitimate use of African American Vernacular English (AAVE).
+
+Simple actions like using dark toned emoji in text or social media posts
+
+## What To Do (instead)
+
+The easiest solution is to pick white people who you identify with and use memes featuring those people instead.
+
+## Further Reading
+
+[https://www.teenvogue.com/story/digital-blackface-reaction-gifs](https://www.teenvogue.com/story/digital-blackface-reaction-gifs)
+"For while reaction GIFs can and do every feeling under the sun, white and nonblack users seem to especially prefer GIFs with black people when it comes to emitting their most exaggerated emotions. Extreme joy, annoyance, anger and occasions for drama and gossip are a magnet for images of black people, especially black femmes."
+
+[https://www.theawl.com/2014/08/memes-and-misogynoir/](https://www.theawl.com/2014/08/memes-and-misogynoir/)
+"On Twitter lie countless handles featuring a Black person’s image, run by users who are most assuredly not Black. These accounts, which often include a “ghetto” name — the formula prefix “La+” is a favored trope — are riddled with poor attempts at Black vernacular, and feature stereotypes from the minstrel stage."
```

Comment:
The further reading section should be in the front matter, formatted like this. It'll mean that you have to remove the quotes, which I might recommend moving into an earlier section, like Examples to help round them out.:
```
reading 
- text: title of the piece
  href: url
```
(It can go anywhere above the --- lines above)